src/vstart.sh

```diff
@@ -480,6 +490,26 @@ EOF
 		COSDSHORT="        osd max object name len = 460
         osd max object namespace len = 64"
 	fi
+        if [ "$objectstore" == "bluestore" ]; then
+                if [ "$spdk_enabled" -eq 1 ]; then
+                        get_nvmeserialnumber
+                        BLUESTORESHORT="        bluestore_block_db_path = \"\"
```

Comment:
s/BLUESTORESHORT/BLUESTORE_OPTS/

the "SHORT" in `COSDSHORT` is  for shorter object name length requirement.

Reply:
Ack. :)